examples/astro/src/pages/web-components.astro

```diff
@@ -32,8 +32,11 @@ import Layout from '../layouts/Layout.astro';
         <ds-dropdown
```

Comment:
dropdown looks weird in my local build

![image](https://github.com/CMSgov/design-system/assets/5159392/21bd617a-b65c-4edd-b369-ec809ee487a4)

Reply:
Did you `yarn build` first?

Reply:
I ran `yarn build:examples` from root before cd'ing into /astro and running `yarn start`